vnext/ReactUWP/Views/PickerViewManager.cpp

```diff
@@ -158,6 +172,7 @@ void PickerShadowNode::RepopulateItems()
         comboboxItem.Foreground(BrushFrom(item["textColor"]));
       comboBoxItems.Append(comboboxItem);
     }
+    m_newItems = true;
```

Comment:
I expected this to be in updateProperties when the "items" property is set. Does it only matter on first creation or is it just that the scenarios you have are always populating before the first render? E.g. what happens if it starts with no items then two items are added? Maybe that already works right?

Reply:
The RepopulateItems method is called from updateProperties when the "items" property is set. Our code clears all the items out and then re-adds them all at once.

Reply:
..oh, to clarify, if there are no items initially the problem doesn't repro...then if the items are added later, the workaround kicks in.

Reply:
Ah, I see that now. Gotta do better with this web UI. I miss CodeFlow ;-). I thought this was in createView.

So, you are always setting it, regardless of count. Probably safest.